br/pkg/streamhelper/basic_lib_for_test.go

```diff
@@ -325,6 +333,11 @@ func (f *fakeCluster) findRegionById(rid uint64) *region {
 	return nil
 }
 
+func (f *fakeCluster) LockRegion(r *region, locks []*txnlock.Lock) *region {
+	r.locks = locks
```

Comment:
I think the locks should be bound to their regions. So the `resolveLocks` can also be implementated in the `fakeCluster` implementation. I'm afraid if we are just record all locks here, we can not take any good from this change.